dockers/docker-fpm-frr/frr/supervisord/critical_processes.j2

```diff
@@ -1,6 +1,9 @@
 program:zebra
 program:staticd
 program:bgpd
+program:vrrpd
```

Comment:
what is the communication channel between vrrpd and zebra, I think we need to put vrrpd into a seperate container, not merging with bgp container. the vrrpd needs to enable/disable separately as feature level. this setup does not allow us to achieve that.